Mathlib/Topology/Algebra/Order/LiminfLimsup.lean

```diff
@@ -6,6 +6,8 @@ Authors: Johannes Hölzl, Mario Carneiro, Yury Kudryashov
 import Mathlib.Algebra.BigOperators.Intervals
 import Mathlib.Algebra.BigOperators.Order
 import Mathlib.Algebra.IndicatorFunction
+import Mathlib.Topology.Algebra.Group.Basic
```

Comment:
Maybe adding the import suggests that the lemmas need to be placed elsewhere?

Reply:
Could you move them to a new file `Mathlib.Topology.Algebra.Group.LiminfLimsup`?

Reply:
Actually no that's probably a bad file. Let's keep them here until someone complains :sweat_smile: